notebook/notebookapp.py

```diff
@@ -1421,7 +1421,11 @@ def init_webapp(self):
             # SSL may be missing, so only import it if it's to be used
             import ssl
             # Disable SSLv3 by default, since its use is discouraged.
```

Comment:
Can you update this accompanying comment to describe what we're now using and why?

Reply:
sure I am going to do that now